tests/common/experimental/test_reducers_and_aggregators.py

```diff
@@ -87,27 +158,27 @@ def test_noop_reducer(self, reducers):
         ],
     )
     def test_min_max_mean_reducers(self, reducer_name, ref, reducers):
-        reduction_shape = (1, 2)
+        reduction_axes = (1, 2)
         input_ = np.arange(-26, 10).reshape((4, 3, 3))
-        for i, red_shape in enumerate([reduction_shape, None]):
-            reducer = reducers[reducer_name](red_shape, False)
-            val = reducer([self.get_nncf_tensor(input_)])
+        for i, red_axes in enumerate([reduction_axes, None]):
```

Comment:
I think some people won't understand why the axes are red.

Reply:
I renamed it